project/LibDependencies.scala

```diff
@@ -9,7 +9,7 @@ object LibDependencies {
 
   // we depend on http-verbs just to integrate via the AuditHooks
   // http calls are made with the underlying play-ws
-  val httpVerbsVersion = "12.0.0"
+  val httpVerbsVersion = "13.0.0-SNAPSHOT"
```

Comment:
Wonder if it makes sense to leave out the snapshot? Might accidentally get merged in